src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -52,10 +54,10 @@ void clear() {
     @Test
     void register_dublicateUser_notOk() {
         User secondUser = new User();
-        secondUser.setLogin(FIRST_LOGIN_OK);
-        secondUser.setPassword(FIRST_PASSWORD_OK);
-        secondUser.setId(FIRST_ID_OK);
-        secondUser.setAge(STANDART_AGE_OK);
+        secondUser.setLogin(STANDARD_LOGIN_OK);
+        secondUser.setPassword(STANDARD_LOGIN_OK_PASSWORD_OK);
+        secondUser.setId(STANDARD_ID_OK);
+        secondUser.setAge(STANDARD_AGE_OK);
```

Comment:
I would create a separate private userConstructor within this class to not repeat code`    private User userConstructor(String login,
                                 String password,
                                 int age) {
        User user = new User();
        user.setLogin(login);
        user.setPassword(password);
        user.setAge(age);
        return user;
    }`